packages/frontend/src/components/common/MigrationBanner.js

```diff
@@ -107,9 +102,36 @@ const CustomButton = styled(FormButton)`
 const IconWrapper = styled.div`
     display: inline;
     margin-right: 10px;
+    margin-left: -10px;
+`;
+
+const CloseButton = styled.button`
+    height: 25px;
+    width: 25px;
+    border: none;
+    margin-left: 30px;
+    cursor: pointer;
+    background-color: transparent;
+    padding: 0;
+
+    @media (max-width: 768px) {
+        margin: 15px auto 0;
+    }
 `;
 
 const MigrationBanner = ({ account, onTransfer }) => {
+    const migrationBannerCloseTime = localStorage.getItem('migrationBannerCloseTime');
```

Comment:
💄 Not a blocker or anything but this would ideally be in the `useEffect` to limit the scope and only access global scope in the place designated for side effects 🙂